packages/vow/src/watch-utils.js

```diff
@@ -191,10 +191,16 @@ export const prepareWatchUtils = (
             // Resolution of the returned vow happened already.
             return;
           }
+          const idToNonStorableResults = provideLazyMap(
```

Comment:
Typing suggestion (unrelated to what the PR is fixing but helping its comprehensibility):
```suggestion
          /** @type {Map<bigint, import('@agoric/internal').TotalMap<number, unknown>>} */
          const idToNonStorableResults = provideLazyMap(
```

Reply:
In the interest of time, I'm merging without this suggestion